### PR TITLE
make kickoff_date not nullable again

### PR DESCRIPTION
**Pull request by gustavomm19**
https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/3988

### CODE REVIEW

File: breathecode/admissions/serializers.py [hunk 1]
```diff
@@ -568,10 +568,6 @@ def validate(self, data):
         ending_date = (data['ending_date'] if 'ending_date' in data else None) or (self.instance.ending_date
                                                                                    if self.instance else None)
 
-        if 'never_ends' in data and data['never_ends'] and kickoff_date:
```
Comment (tommygonzaleza) [inline breathecode/admissions/serializers.py:571]: Debes agregar esta validacion pero para 'ending_date'

**Review by tommygonzaleza — CHANGES_REQUESTED** [review]

**Comment by jefer94** [comment]
Add test case where you is trying to set `kickoff_date` as `null` and see the result

**Review by jefer94 — APPROVED** [review]
